docs/config/README.md

```diff
@@ -61,17 +61,20 @@ This is the list of prompt-wide configuration options.
 
 ### Options
 
-| Variable       | Default                       | Description                                            |
-| -------------- | ----------------------------- | ------------------------------------------------------ |
-| `add_newline`  | `true`                        | Add a new line before the start of the prompt.         |
-| `prompt_order` | [link](#default-prompt-order) | Configure the order in which the prompt module occurs. |
-| `scan_timeout` | `30`                          | Timeout for starship to scan files (in milliseconds).  |
+| Variable                | Default                       | Description                                                  |
+| ----------------------- | ----------------------------- | ------------------------------------------------------------ |
+| `clear_before_printing` | `false`                       | Clear the terminal to the bottom before printing the prompt. |
```

Comment:
Would there be any reason for users to set this to `true`?
I can't think of cases where this configuration may be useful.

Reply:
I can't think of any. My thought was mainly not to potentially break someone's prompt by accident.

Reply:
I'd be in favor of having `clear_before_printing` be the default behavior without configuration, at least until we find out it is needed. I have done some local testing in each of the supported shells and haven't run into any issues.

I think it will be otherwise difficult to explain the value of this configuration option to users, and it will just end up being a suggested fix to the rare user that reports a bug on GitHub.

Do you think that makes sense?

Reply:
Yes.